.github/workflows/linters.yml

```diff
@@ -0,0 +1,19 @@
+name: Linters
+
+on: pull_request
+
+jobs:
+  rubocop:
+    name: Rubocop
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - name: Setup Rubocop
+        run: |
+          gem install --no-document rubocop:'~>0.81.0' # https://docs.rubocop.org/en/stable/installation/
+          [ -f .rubocop.yml ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ruby/.rubocop.yml
+      - name: Rubocop Report
+        run: rubocop --color

```

Comment:
[Required] This config file you used is erroneous, you should have use [this one](https://github.com/microverseinc/linters-config/blob/master/ror/.github/workflows/linters.yml) from microverse ror projects linters config files.